base/replutil.jl

```diff
@@ -340,15 +340,18 @@ function show_backtrace(io::IO, t, set=1:typemax(Int))
     # we may not declare :eval_user_input
     # directly so that we get a compile error
     # in case its name changes in the future
-    show_backtrace(io, try
-                         symbol(string(eval_user_input))
-                       catch
-                         :(:) #for when client.jl is not yet defined
-                       end, t, set)
+    process_entry(lastname, lastfile, lastline, n) = show_trace_entry(io, lastname, lastfile, lastline, n)
+
+    process_backtrace(process_entry,
+                        try
+                            symbol(string(eval_user_input))
+                        catch
+                            :(:) #for when client.jl is not yet defined
+                        end, t, set)
 end
 
-# show the backtrace, up to (but not including) top_function
-function show_backtrace(io::IO, top_function::Symbol, t, set)
```

Comment:
Note that the elimination of this `show_backtrace` broke IJulia.   What is the equivalent now of `show_backtrace(io, top, t, set)`?